### PR TITLE
Upgrade raven to 7.6.0, add environment and release options

### PR DESCRIPTION
**Pull request by mickeyreiss**
Hi @stve,

I'd like to use a few features in `raven-java` that have been introduced since version 7.0.0. 

Can you accept this patch and push a release to maven?

Let me know if you'd like me to improve upon anything. (I noticed inconsistent code style, so I wasn't sure which styles were preferred in this codebase. As such, I'd recommend reviewing this diff with `?w=1`.)

Thanks! 😸

### CODE REVIEW

**Comment by stve** [comment]
hi @mickeyreiss, this change looks good to me. Let's not worry about the formatting issues, i'll straighten that out post-merge.

I do have one request, can you add null checks where `environment` and `release` are set on the `SentryAppender` since those are not required configuration values and shouldn't be passed unless set. Thanks!

**Comment by mickeyreiss** [comment]
@stve Thanks for the review. I rebased the change you suggested. Here's the incremental diff:

``` diff
diff --git a/src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java b/src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java
index 9833f9d..b7ed620 100644
--- a/src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java
+++ b/src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java
@@ -63,8 +63,12 @@ public class RavenAppenderFactory extends AbstractAppenderFactory {
    appender.setName("dropwizard-raven");
    appender.setContext(context);
    appender.setDsn(dsn);
-        appender.setRelease(release);
-        appender.setEnvironment(environment);
+   if (release != null) {
+       appender.setRelease(release);
+   }
+   if (tags != null) {
+       appender.setEnvironment(environment);
+   }
    if(tags != null)
        appender.setTags(tags);
```

File: src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java [hunk 1]
```diff
@@ -42,41 +47,55 @@ public void setTags(String tags) {
...
+	checkNotNull(context);
+
+	final SentryAppender appender = new SentryAppender();
+	appender.setName("dropwizard-raven");
+	appender.setContext(context);
+	appender.setDsn(dsn);
+	if (release != null) {
+		appender.setRelease(release);
+	}
+	if (tags != null) {
```
Comment (stve) [inline src/main/java/com/tradier/raven/logging/RavenAppenderFactory.java]: this should be `if(environment != null) {`

**Comment by stve** [comment]
Looks good, i left one small comment in the diff

**Comment by mickeyreiss** [comment]
Done. Sorry about the sloppy mistake!

**Comment by stve** [comment]
👍  looks great, thanks! I'll have a new release out in the next week or so with these changes.

**Comment by mickeyreiss** [comment]
@stve Are there any blockers to the next release? Anything I can do to help?

**Comment by stve** [comment]
hey @mickeyreiss, thanks for the nudge, apologies that this took a few days. I've just released `0.9.2` which includes your changes. I don't believe it has synced to the central repository yet, but would expect it to shortly.

**Pull request merged**